### PR TITLE
Canada Employment Amount

### LINKED ISSUE #322: Canada Employment Amount
reference: https://www.canada.ca/en/revenue-agency/services/tax/individuals/topics/about-your-tax-return/tax-return/completing-a-tax-return/deductions-credits-expenses/line-31260-canada-employment-amount.html

https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/5011-r/5011-r-22e.pdf

### PR DESCRIPTION
**Pull request by sqw0418**
Fixes #322

### CODE REVIEW

**Review by PavelMakarchuk — APPROVED** [review]

File: policyengine_canada/variables/gov/cra/tax/income/credits/canada_employment_amount.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    unit = CAD
+    definition_period = YEAR
+    reference = "https://www.canada.ca/en/revenue-agency/services/tax/individuals/topics/about-your-tax-return/tax-return/completing-a-tax-return/deductions-credits-expenses/line-31260-canada-employment-amount.html"
+
+    def formula(person, period, parameters):
+        max_amount = parameters(
+            period
+        ).gov.cra.tax.income.credits.canada_employment_amount.max_amount
+
+        employment_income = person("employment_income", period)
```
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/cra/tax/income/credits/canada_employment_amount.py:17]: Make an `income_sources` parameter file with employment_income and other_employment_income
```suggestion
        countable_income = add(person, period, p.income_sources)
```

File: policyengine_canada/variables/gov/cra/tax/income/credits/canada_employment_amount.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+class canada_employment_amount(Variable):
+    value_type = float
+    entity = Person
+    label = "Canada Employment Amount"
+    unit = CAD
+    definition_period = YEAR
+    reference = "https://www.canada.ca/en/revenue-agency/services/tax/individuals/topics/about-your-tax-return/tax-return/completing-a-tax-return/deductions-credits-expenses/line-31260-canada-employment-amount.html"
+
+    def formula(person, period, parameters):
+        max_amount = parameters(
```
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/cra/tax/income/credits/canada_employment_amount.py:13]: and remove `.max_amount`
```suggestion
        p = parameters(
```

File: policyengine_canada/variables/gov/cra/tax/income/credits/canada_employment_amount.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+            period
+        ).gov.cra.tax.income.credits.canada_employment_amount.max_amount
+
+        employment_income = person("employment_income", period)
+        other_employment_income = person("other_employment_income", period)
+        total_income = employment_income + other_employment_income
+
```
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/cra/tax/income/credits/canada_employment_amount.py:20]:
```suggestion
        # While the government website states that the Canada Employment Amount is designed
        # to support workers with work-related expenses, those expenses are not part of the formula.
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

File: policyengine_canada/parameters/gov/cra/tax/income/credits/canada_employment_amount/max_amount.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+description: Canada provides the following maximum amount under the canada employment amount. 
+values:
+  2022-01-01: 1_287
+metadata:
+  unit: currency-CAD
+  label: Canada Employment Amount max amount
+  reference:
+  - title: Government of Canada, Line 31260 – Canada employment amount
+    href: https://www.canada.ca/en/revenue-agency/services/tax/individuals/topics/about-your-tax-return/tax-return/completing-a-tax-return/deductions-credits-expenses/line-31260-canada-employment-amount.html
+  - title: Canada Income Tax Act, 118 (10) - Canada employment credit
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/cra/tax/income/credits/canada_employment_amount/max_amount.yaml:10]:
```suggestion
  # The law specifies $1,000.
  - title: Canada Income Tax Act, 118 (10) - Canada employment credit
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

File: policyengine_canada/parameters/gov/cra/tax/income/credits/canada_employment_amount/max_amount.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+description: Canada provides the following maximum amount under the Canada employment amount. 
+
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/cra/tax/income/credits/canada_employment_amount/max_amount.yaml:2]:
```suggestion
values:
```

File: policyengine_canada/parameters/gov/cra/tax/income/credits/canada_employment_amount/income_sources.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+description: Canada considers the following income sources when computing the employment amount credit.
+lues:
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/cra/tax/income/credits/canada_employment_amount/income_sources.yaml:2]:
```suggestion
values:
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**